client-tests/jvm/build.gradle

```diff
@@ -17,13 +17,6 @@ apply plugin: 'kotlin'
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
-compileTestKotlin {
```

Comment:
Why was this removed? The point of the client-tests module is to ensure backwards compatibility with projects running on Kotlin 1.3.*.

I see the point in removing the same block in other modules, but I thinks it should be kept in client-tests.

Reply:
Okay, I thought it was just an empty configuration block.